bot/src/slots.ts

```diff
@@ -17,11 +20,19 @@ export const slots = [
   },
   {
     name: 'QUANTITY',
-    query: () => { return 'How many items would you like?' }
+    query: () => { return 'How many items?' },
+    retry: () => { return 'Please enter how many items.' },
+    validate: (submittedValue) => {
+      const num = wordsToNum(submittedValue);
+      if ( isNaN(num) ) {
+        return { isValid: false, reason: 'unable to parse' }
+      }
+      return { isValid: true, reason: null }
+    }
   },
   {
     name: 'TARGET',
-    query: () => { return 'What item would you like to target?' }
+    query: () => { return 'What item would you like to replace?' }
```

Comment:
I know this was existing and for uniformity we should keep this..

but just an fyi short hand tip:
`query: () => 'What item would you like to replace?'